VnE/Source/MainWindow.py

```
@@ -312,6 +354,85 @@ def setModel(self, model):
+    def _browse_file(self):
+        """Открывает диалог сохранения файла и обновляет поле пути."""
+        file_path, _ = QtWidgets.QFileDialog.getSaveFileName(
+            self,
+            "Save image",
+            "",
+            "Image (*.png *.tiff)"
+        )
+        if file_path:
+            self.file_path_edit.setText(file_path)
+
+    def _on_accept(self):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_on_accept`

Add return type annotation: `None`

(ANN202)